From a5d210deb9dd9b60260b877ed37af3d417f872b6 Mon Sep 17 00:00:00 2001
From: Matthew Fatheree <Matthew.Fatheree@belkin.com>
Date: Sun, 4 May 2014 01:23:17 +0700
Subject: [PATCH 10/34] Add linksys-base-files to package

Add linksys-base-files to ./package. This package contains scripts and
files that specific for Linksys Mamba WRT1900AC

Signed-off-by: Matthew Fatheree <Matthew.Fatheree@belkin.com>
---
 package/linksys-base-files/Makefile              |   44 ++++++++++++++++++++++
 target/linux/mvebu/profiles/101-Linksys-Mamba.mk |    3 +-
 2 files changed, 46 insertions(+), 1 deletion(-)
 create mode 100644 package/linksys-base-files/Makefile

diff --git a/package/linksys-base-files/Makefile b/package/linksys-base-files/Makefile
new file mode 100644
index 0000000..9e67f74
--- /dev/null
+++ b/package/linksys-base-files/Makefile
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2007-2013 OpenWrt.org
+# Copyright (C) 2010 Vertical Communications
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/version.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=linksys-base-files
+PKG_RELEASE:=2
+REVISION:=0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/linksys-base-files
+  SECTION:=base
+  CATEGORY:=Base system
+  TITLE:=Linksys Base filesystem for OpenWrt
+  URL:=http://openwrt.org/
+  VERSION:=$(PKG_RELEASE)-$(REVISION)
+endef
+
+define Package/base-files/description
+ This package contains the Linksys base filesystem and system scripts for OpenWrt.
+endef
+
+define Build/Prepare
+	mkdir -p $(PKG_BUILD_DIR)
+endef
+
+define Build/Compile/Default
+
+endef
+Build/Compile = $(Build/Compile/Default)
+
+define Package/linksys-base-files/install
+	$(CP) ./files/* $(1)/
+endef
+
+$(eval $(call BuildPackage,linksys-base-files))
diff --git a/target/linux/mvebu/profiles/101-Linksys-Mamba.mk b/target/linux/mvebu/profiles/101-Linksys-Mamba.mk
index de048a6..b78e4f9 100644
--- a/target/linux/mvebu/profiles/101-Linksys-Mamba.mk
+++ b/target/linux/mvebu/profiles/101-Linksys-Mamba.mk
@@ -11,7 +11,8 @@ define Profile/Mamba
 	kmod-usb-storage \
 	kmod-of-i2c kmod-i2c-core kmod-i2c-mv64xxx \
 	kmod-ata-core kmod-ata-marvell-sata \
-	kmod-rtc-marvell kmod-thermal-armada
+	kmod-rtc-marvell kmod-thermal-armada \
+	linksys-base-files
 endef
 
 define Profile/Mamba/Description
-- 
1.7.9.5

